sdk/schemaregistry/azure-schemaregistry/setup.py

```diff
@@ -13,7 +13,7 @@
 
 # Change the PACKAGE_NAME only to change folder and different name
 PACKAGE_NAME = "azure-schemaregistry"
-PACKAGE_PPRINT_NAME = "Schema Registry"
+PACKAGE_PPRINT_NAME = "Azure Schema Registry"
```

Comment:
do you want this to change here? Do you need to change the naming in your tsp-location file?

Reply:
Yes, we want this change. So I'm actually trying to figure out the best way to do this rn + responding to Yuchao. But if setup.py is not overwritten (which it may not be b/c we need the extras_require), we want to add this in. If it is overwritten by the generated code, this change will be made anyway.